### PR TITLE
Booklist classes

### PR DESCRIPTION
**Pull request by SamuelTakyiBoateng**
## Milestone 2
- We have Create class methods to add and remove books.
- We can add and remove books 
- We add classes to the project

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Samuel, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not contributed by all (2) assigned pair group members. Kindly ensure that all assigned members to this project contribute evenly to this task, and then request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by HenaryKhuraijam** [comment]
Thank you for your support.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+function appendBooklist(book) {
+  const articleBooklist = document.getElementById('booklist');
+  articleBooklist.className = 'booklist';
+  articleBooklist.id = 'booklist';
+  articleBooklist.appendChild(compileBookArticle(book));
+  document.getElementById(`id-${book.id}`).addEventListener('click', () => {
+    removeBooks(`id-${book.id}`);
+  });
+}
+
```
Comment (rloterh) [inline index.js]: - [ ] In this activity we require that all objects, array or similar, are placed inside of a class. Let's, therefore, ensure to refactor the functions and arrays into class(es), and let's ensure app works correctly after refactor.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Henary Singh & Samuel, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🌴 🚦 🚗 🏃 🏫 🏡 _🏃

## Highlights 👍  
-  Good job so far on implementing classes in your code!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Generally, it's a goof idea that the subheading text _Author1_ and _Author2_ or similar are replaced with your respective full names in the ReadMe.
Implementing the above change(s) will make your ReadMe/repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (HenaryKhuraijam) [reply index.js]: Thank you for finding out my mistakes.

**Pull request merged**